pkg/arvo/lib/hark/graph-hook.hoon

```diff
@@ -0,0 +1,63 @@
+/-  sur=hark-graph-hook
+/+  graph-store, resource
+^?
+=<  [. sur]
+=,  sur
+|%
+::
+++  dejs
+  =,  dejs:format
+  |%
+  ::
+  ++  graph-indices
+    %-  ot
+    :~  graph+dejs-path:resource
+        indices+(as graph-index)
+    ==
+  ::
+  ++  graph-index
+    ^-  $-(json index:graph-store)
+    (su ;~(pfix net (more net dem)))
+  ::
+  ++  action
+    %-  of
+    :~  listen+dejs-path:resource
+        ignore+dejs-path:resource
+        set-mentions+bo
+        set-watch-on-self+bo
+    ==
+  --
+++  enjs
+  =,  enjs:format
+  |%
+  ::
```

Comment:
unnecessary. `::` does not go at the beginning or end of a core definition, but it goes in between every arm definition within the core, as per our style guide.